server/src/domain/library/library.service.ts

```diff
@@ -129,6 +187,10 @@ export class LibraryService {
       isVisible: dto.isVisible ?? true,
     });
 
+    if (dto.type === LibraryType.EXTERNAL && library.watched) {
+      await this.watch(library);
+    }
```

Comment:
This ends up starting the watcher somewhat ad-hoc in the server container, I believe. Would it be better to trigger a config refresh or such that asynchronously re-evaluates the library watching (ie what happens in `init()`)?